tests/eventEmitterListenerCount.test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable max-len */
+
+'use strict';
+
+const { EventEmitter } = require('../src/MyEventEmitter');
+
+describe('EventEmitter "listenerCount" method tests', () => {
```

Comment:
The final message will be

```
... tests should ...???
```
it is better to call the block just `"listenerCount" method`
the same for all the other describes